generators/app/index.js

```diff
@@ -19,48 +18,58 @@ module.exports = fountain.Base.extend({
             jasmine: true
           }
         }
-      };
+      });
+
+      if (this.props.framework === 'react') {
+        this.mergeJson('package.json', {
+          devDependencies: {
+            'babel-preset-react': '^6.1.18',
```

Comment:
Not sure, doing ESLint in React is not better than doing React in ESLint...

As I see it, I would prefer having ESLint React related configuration inside generator ESLint.